From 8daf63ab883e77b5623b8951538c5d8a37a42930 Mon Sep 17 00:00:00 2001
From: Jannick <thirdedition@gmx.net>
Date: Mon, 10 May 2021 00:18:28 +0200
Subject: [PATCH] igraph.pc.in: add member 'Cflags.private'

Flag 'Cflags.private' is an extension of 'pkgconfig'
(https://github.com/pkgconf/pkgconf) over 'pkg-config'
(https://www.freedesktop.org/wiki/Software/pkg-config/).
'Cflags.private' is additionally emitted when calling
'pkg-config --static --cflags'.
---
 igraph.pc.in | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/igraph.pc.in b/igraph.pc.in
index 4c036fbdc..e755833d7 100644
--- a/igraph.pc.in
+++ b/igraph.pc.in
@@ -10,3 +10,9 @@ URL: @PROJECT_HOMEPAGE_URL@
 Libs: -L${libdir} -ligraph
 Libs.private: @PKGCONFIG_LIBS_PRIVATE@
 Cflags: -I${includedir}/igraph
+# Flag 'Cflags.private' is an extension of 'pkgconfig'
+# (https://github.com/pkgconf/pkgconf) over 'pkg-config'
+# (https://www.freedesktop.org/wiki/Software/pkg-config/).
+# 'Cflags.private' is additionally emitted when calling
+# 'pkg-config --static --cflags'.
+Cflags.private: -DIGRAPH_STATIC=1
-- 
2.31.1.windows.1

